### PR TITLE
Reformat repo

### PR DESCRIPTION
**Pull request by seb231**
Model workspace, catalog, library and functions + tests (structure and acceptance)
The model runs so far but passing empty data around.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+*~
```
Comment (seb231) [inline .gitignore]: I never remember if you're supposed to include the .gitignore! thanks!

File: src/kixi/gwyn/gwyn.clj [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+(defworkflowfn join-historical-and-new-scores-1-0-0
+  {:witan/name :fire-risk/join-historical-and-new-scores
+   :witan/version "1.0.0"
+   :witan/input-schema {:commercial-properties-with-scores sc/CommercialPropertiesWithScores
+                        :historical-fire-risk-scores sc/HistoricalFireRiskScores}
+   :witan/output-schema {:historical-and-new-scores sc/HistoricalFireRiskScores}}
+  [{:keys [commercial-properties-with-scores historical-fire-risk-scores]} _]
+  {:historical-and-new-scores {}})
+
+(defworkflowfn update-score-with-priority-1-0-0
```
Comment (seb231) [inline src/kixi/gwyn/gwyn.clj]: Maybe just 'update-score'? with priority doesn't really make sense as they are already prioritised

File: src/kixi/gwyn/schemas.clj [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  [col-schema]
+  (mapv (fn [s] (let [datatype (:schema s)
+                      fieldname (:name s)]
+                  (s/one datatype fieldname)))
+        (:column-names col-schema)))
+
+;; (def FireStations
+;;   (make-ordered-ds-schema [[:station s/Str] [:area_km2 java.lang.Double]
+;;                            [:radius java.lang.Double] [:lat java.lang.Double]
+;;                            [:long java.lang.Double]]))
```
Comment (seb231) [inline src/kixi/gwyn/schemas.clj]: How come you commented this out?

Comment (Eleonore9) [inline src/kixi/gwyn/schemas.clj]: I'm not using data for now, just empty maps.
We'll need this schema, but it'd lead to an error atm

File: src/kixi/gwyn/gwyn.clj (hunk 2 above)
Comment (Eleonore9) [inline src/kixi/gwyn/gwyn.clj]: Sure, I'll change that!

**Comment by Eleonore9** [comment]
@seb231: I started Witan names with `:fire-risk/` instead of `:gwyn/` thinking it'll be easier to identify which model it's part of when Antony integrate in Witan for example. Wdyt?

File: src/kixi/gwyn/schemas.clj (hunk 3 above)
Comment (seb231) [inline src/kixi/gwyn/schemas.clj]: 👍

**Comment by seb231** [comment]
:shipit:

**Pull request merged**